tests/test_interconnect/test_interconnect_cgra.py

```diff
@@ -13,6 +10,7 @@
 import magma
 from memory_core.memory_core_magma import config_mem_tile
 from collections import deque
+from memory_core.memtile_util import ONYX_PORT_REMAP
```

Comment:
Is the ONYX_PORT_REMAP code technology specific? I.e. does the ONYX_PORT_REMAP code in this file need some kind of "if (GF) ... else if (TSMC) ..." wrapper?

Reply:
No it doesn't